pkg/node/manager/manager.go

```diff
@@ -583,9 +586,17 @@ func (m *Manager) upsertIntoIDMD(prefix netip.Prefix, id identity.NumericIdentit
 	}
 }
 
+func (m *Manager) removeFromIDMD(prefix netip.Prefix, id identity.NumericIdentity, rid ipcacheTypes.ResourceID) {
+	if id == identity.ReservedIdentityHost {
```

Comment:
FYI: ipcache.RemoveLabels() doesn't actually care about the contents of the labels; it just removes all labels for that prefix+resource.

Reply:
In other words `m.ipcache.RemoveLabels(prefix, labels.Labels{}, rid)` will also accomplish what you want.

Reply:
I've tried to keep this as close as possible to #27406, so I've only changed `RemoveMetadata` -> `RemoveLabels` but happy to change to just ipcache.RemoveLabels.

Reply:
I think keeping it as close as possible makes sense. I'll resolve this and merge the PR as it's ready to merge and is a release blocker.